addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -465,7 +465,7 @@ public void redact(ScriptNode node, String replace, String replaceWith, boolean
         this.refreshMessage();
     }
 
-    public void perameterize(
+    public void parametrize(
```

Comment:
Why not just fix the typo? No need to introduce other variant (e.g. `ZestParameterizeDialog`).

Reply:
I'll have a look.

Reply:
Took me a minute to clue into the exact issue here, but I agree it should have an e after the t.